src/BlackSlope.Api/BlackSlope.Api.csproj

```diff
@@ -39,6 +38,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.4" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.13" />

```

Comment:
Linter was really spazzing about conflicting versions. Did a harder ref to the latest 3.x package.